gromacs/config.py

```diff
@@ -266,50 +228,36 @@
 """
 from __future__ import absolute_import, with_statement
 
-import os, errno, subprocess
-from ConfigParser import SafeConfigParser
+import os
+import logging
+import subprocess
 
+from ConfigParser import SafeConfigParser
 from pkg_resources import resource_filename, resource_listdir
 
 from . import utilities
 
-# Defaults
-# --------
-# hard-coded package defaults
 
-#: name of the global configuration file.
+# Default name of the global configuration file.
 CONFIGNAME = os.path.expanduser(os.path.join("~",".gromacswrapper.cfg"))
 
-#: Holds the default values for important file and directory locations.
-#:
-#: :data:`configdir`
-#:    Directory to store user templates and configurations.
-#:    The default value is ``~/.gromacswrapper``.
-#: :data:`qscriptdir`
-#:    Directory to store user supplied queuing system scripts as
-#:    used by :mod:`gromacs.qsub`.
-#:    The default value is ``~/.gromacswrapper/qscripts``.
-#: :data:`templatesdir`
-#:    Directory to store user supplied template files such as mdp files.
-#:    The default value is ``~/.gromacswrapper/templates``.
-#: :data:`managerdir`
-#:    Directory to store configuration files for different queuing system
-#:    managers as used in :mod:`gromacs.manager`.
-#:    The default value is ``~/.gromacswrapper/managers``.
+# More default values
```

Comment:
I don't like deleting documentation that is still correct. It's quite a lot of work to write it in the first place an I'm always grateful when I find variables properly documented. Unlike code comments it shows tht someone thought about how to communicate to a user or another developer and not just to oneself. 

--
Oliver Beckstein
email: orbeckst@gmail.com


> Am Jul 26, 2016 um 16:44 schrieb Pedro Lacerda <notifications@github.com>:
> 
> Because is developer documentation quickly implied by variable names, except qscriptdir which again can be found just searching the project.